app/controllers/roles_controller.rb

```diff
@@ -52,6 +52,8 @@ def edit
   end
 
   def update
+    @role.set_taxonomies(taxonomy_params!)
```

Comment:
Ditto, no reason why regular `.update_attributes` couldn't handle this -the model- is the one who handles it